docs/guides/dns/dnscrypt-proxy.md

```diff
@@ -6,13 +6,15 @@ As an alternative tool to this end, consider [cloudflared](https://github.com/cl
 
 ### Installing `dnscrypt-proxy`
 
-Raspberry Pi OS and Debian as well as Ubuntu come with packages for `dnscrypt-proxy`, which makes its installation a breeze:
+Under Debian 13 `Trixie` and Ubuntu 25 `Plucky Puffin` official packages are available and therefore can be installed with the following commands:
```

Comment:
Instead of incrementing the versions every time the versions change why not just link to https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-linux#installation-on-linux-overview and let users figure out how to install on their system?

Reply:
I agree, the `deb` installation should be preferred. They mention it under this section.

<img width="2122" height="768" alt="Image" src="https://github.com/user-attachments/assets/b500e826-e56b-421b-97bd-512c48721bb6" />

___

I think the main issue I see here is that you upgraded the instructions to Trixie and Ubuntu 25 and it sounds like this command does not work with older Debian/Ubuntu versions. But this is not correct.

Reply:
By default, the instructions listed there will not work on Raspberry Pi OS, due to the testing repo not being added.

The instructions also reference using the testing or unstable package, the ones I think I am referencing by saying Trixie or Puffin (the docs are a little out of date).

In Debian 12 and derivatives, there is not a  dnscrypt-proxy package in the official repo  (tested on Raspberry Pi OS and Debian 12).

Should the docs point the user towards using the "unstable" (now stable for oldstable debian 12) or tell the user to compile since a lot of people are still using Debian 12, including raspberry pi os as of writing.

Reply:
Yes, it is.

```suggestion
Some distributions provide official packages, which can be installed using the commands (or their equivalent):
```

Reply:
What do you think about this suggestion and the other?